component/libfuse/libfuse2_handler.go

```diff
@@ -1,4 +1,4 @@
-package libfuse
+//go:build windows
```

Comment:
is libfuse2 windows only? What are the distinctions between libfuse2_handler and libfuse_handler having separate go:build os specifications?